### PR TITLE
Minor cleanups to GL passthrough function handling. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: src/library_webgl.js [hunk 1]
```diff
@@ -4207,7 +4206,7 @@ function createGLPassthroughFunctions(lib, funcs) {
       let cName = name;
       if (name.includes('[')) {
         cName = name.replace('[', '').replace(']', '');
-        name = cName.substr(0, cName.length-1);
+        name = cName.slice(0, -1)
```
Comment (juj) [inline src/library_webgl.js:4209]: Removing semicolons at the end seems nonidiomatic JS, here and above?

File: src/library_webgl.js [hunk 2]
```diff
@@ -4194,11 +4194,10 @@ function createGLPassthroughFunctions(lib, funcs) {
     const returnStub = '(function(' + args + ') { return GLctx[\'NAME\'](' + args + ') })';
     const sigEnd = range(num).map(() => 'i').join('');
     names.split(' ').forEach((name) => {
```
Comment (juj) [inline src/library_webgl.js:4196]: Maybe the split could be removed here, and directly have the above thing contain arrays, e.g. `[0, ['finish', 'flush']]`?

(Could be a separate improvement, or leave out, lgtm either way)

File: src/library_webgl.js (hunk 1 above)
Comment (sbc100) [reply src/library_webgl.js:4209]: Oops, didn't mean to do that.

File: src/library_webgl2.js [hunk 3]
```diff
@@ -1116,28 +1116,28 @@ var LibraryWebGL2 = {
...
- [3, ''],
- [4, 'clearBufferfi'],
- [5, 'vertexAttribI4i vertexAttribI4ui copyBufferSubData texStorage2D renderbufferStorageMultisample'],
- [6, 'texStorage3D'],
- [7, ''],
- [8, ''],
- [9, 'copyTexSubImage3D'],
- [10, 'blitFramebuffer']];
-
 #if MAX_WEBGL_VERSION >= 2
```
Comment (kripken) [inline src/library_webgl2.js:1119]: This entire library is only included from `modules.js` if that condition is true, so I don't think we need it here.

Comment (sbc100) [reply src/library_webgl2.js:1119]: I think maybe you are right, but I'm not 100% sure.   Maybe a followup?

Comment (kripken) [reply src/library_webgl2.js:1119]: Sounds good.

**Review by kripken — APPROVED** [review]

**Pull request merged**